src/containers/CallIn/ThankYou/ThankYou.js

```diff
@@ -38,6 +42,8 @@ const ColorContentRow = ({ bg, children}) => (
 class ThankYou extends Component {
 
     state = {
+        calledState: null,
```

Comment:
Seems like these don't need to be added to state. Seems like they could just be passed to the new `reportCall` function as arguments.